numba_cuda/numba/cuda/tests/cudadrv/test_linker.py

```diff
@@ -329,6 +329,20 @@ def test_get_local_mem_per_specialized(self):
         calc_size = np.dtype(np.float64).itemsize * LMEM_SIZE
         self.assertGreaterEqual(local_mem_size, calc_size)
 
+    def test_link_for_different_cc(self):
+        if not config.CUDA_USE_NVIDIA_BINDING:
+            self.skipTest("Ctypes Linker does not support outputting PTX.")
```

Comment:
I tried using cuobjdump to get PTX. But it requires the output to have proper fatbin binary headers. The output of cuLink is cubin binary and cannot result in PTX.

Reply:
It's not possible to run this test with the ctypes linker because it doesn't have a way to do LTO, and I don't think there's a way we could reasonably implement that and have it produce LTO-d PTX - so there's not much we can check here for the ctypes linker.